catlin/pkg/consts/consts.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package consts
+
+const (
+	VersionLabel                 = "app.kubernetes.io/version"
+	DisplayNameAnnotation        = "tekton.dev/displayName"
+	MinPipelineVersionAnnotation = "tekton.dev/pipelines.minVersion"
+	TagsAnnotation               = "tekton.dev/tags"
```

Comment:
I see right now Catlin is checking these labels/annotation 
But `Catlin` don't have information about optional and mandatory labels/annotation

Just a suggestion like can we add doc somewhere to give just a brief about the `Catlin` and what all things it validates something like that
But ya if you planned documentation sometime later as part of another PR that's totally fine

Reply:
@savitaashture these labels and annotations are documented in the TEP  - https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md 
which is what it tries to validate against.

Reply:
+1 on adding docs, it's fine to do so in a follow-up PR, but it would be nice to have at least a basic readme.